partner_contact_birthdate/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2014-2015  Grupo ESOC <www.grupoesoc.es>
+# © 2017-Apertoso N.V. (<http://www.apertoso.be>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Contact's birthdate",
+    'version': '11.0.1.0.0',
+    "author": "Tecnativa,"
+              "Odoo Community Association (OCA)",
+    "category": "Customer Relationship Management",
+    "website": "https://www.tecnativa.com/",
```

Comment:
@Olivier-LAURENT IMHO Please change URL as described in  https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10

Reply:
@Olivier-LAURENT @pedrobaeza could this module merged?